MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -432,6 +517,28 @@ - (void)processCrashReport {
   }
 }
 
+- (void)processLogBufferAfterCrash {
+  NSArray *files = [[NSFileManager defaultManager] contentsOfDirectoryAtPath:self.logBufferDir
+                                                                       error:NULL];
+  for (NSString *tmp in files) {
+    if ([[tmp pathExtension] isEqualToString:kMSLogBufferFileExtension]) {
+      NSString *filePath = [self.logBufferDir stringByAppendingPathComponent:tmp];
+      NSData *serializedLog = [NSData dataWithContentsOfFile:filePath];
+
+      id <MSLog> item = [NSKeyedUnarchiver unarchiveObjectWithData:serializedLog];
+      if (item) {
+        if ([((NSObject *) item) isKindOfClass:[MSAppleErrorLog class]]) {
```

Comment:
The problem is that crashes doesn't know about other types of logs at all. The only reliable way that I can think of right now would be to introduce a priority-property on MSLog, which I think is out of scope for now.